app/models/wrangling_guideline.rb

```diff
@@ -0,0 +1,14 @@
+class WranglingGuideline < ActiveRecord::Base
+  acts_as_list
+
+  attr_protected :content_sanitizer_version
+
+  validates_presence_of :content, :title
+  validates_length_of :content, :maximum => ArchiveConfig.CONTENT_MAX,
+    :too_long => ts('cannot be more than %{max} characters long.', :max => ArchiveConfig.CONTENT_MAX)
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.